website/docs/tutorials/guides/testing.md

```diff
@@ -825,6 +825,75 @@ There is no type checking for `variable_name`, `key`, `variable_type` make sure
 `key` is a list of arguments because cairo `@storage_var` maps any number of felt arguments to any number of felt values
 :::
 
+### `reflect`
+```python
+def reflect(self, value: Union[VmConstsReference, RelocatableValue, int]) -> Union[NamedTuple, RelocatableValue, int]:
+```
+Converts Cairo object into Python `NamedTuple` (complex structure) or keeps it a simple type `RelocatableValue` (pointer) or `int` (felt). It can be used to easily print and compare complex structures.
+```cairo title="./test/example_test.cairo"
```

Comment:
Personally I don't think that's a good idea - it's a comprehensive example that shows all of the available features.

Reply:
> it's a comprehensive example that shows all of the available features.

And that's great that this is comprehensive. But it would be even better if this were a comprehensive **set** of examples **with a commentary of what's going on here**.

Reply:
+1 This is very hard to read

Reply:
I don't think there's a much better way to show nested comparisons than this. Without that we would need a lot of boilerplate to show even the simplest things.